src/model/points-model.js

```diff
@@ -81,27 +81,36 @@ export default class PointsModel extends Observable{
     }
   };
 
-  addPoint = (updateType, update) => {
-    this.#points = [
-      update,
-      ...this.#points,
-    ];
+  addPoint = async (updateType, update) => {
+    try {
+      const response = await this.#pointsApiService.addPoint(update);
+      const newPoint = this.#adaptToClient(response);
 
-    this._notify(updateType, update);
+      this.#points = [
+        newPoint,
+        ...this.#points,
+      ];
+      this._notify(updateType, newPoint);
+    } catch(err) {
+      throw new Error('Can\'t add point');
```

Comment:
Чтобы ускорять дебаг в том числе и для других программистов, лучше добавлять сразу сам объект на котором произошла ошибка
```suggestion
      throw new Error(`Can't add point, update: ${update}`);
```